code/datums/brain_damage/severe.dm

```diff
@@ -0,0 +1,219 @@
+		if((istype(M, /mob/living/simple_animal)) || M.ckey)
+			return FALSE
+	return TRUE
+
+/datum/brain_trauma/severe/monophobia/proc/stress_reaction()
+	if(owner.stat != CONSCIOUS)
+		return
+
+	var/high_stress = (stress > 60) //things get psychosomatic from here on
+	switch(rand(1,6))
+		if(1)
+			if(!high_stress)
+				to_chat(owner, "<span class='warning'>You feel sick...</span>")
+			else
+				to_chat(owner, "<span class='warning'>You feel really sick at the thought of being alone!</span>")
+			addtimer(CALLBACK(owner, /mob/living/carbon.proc/vomit, high_stress), 50) //blood vomit if high stress
+		if(2)
+			if(!high_stress)
+				to_chat(owner, "<span class='warning'>You can't stop shaking...</span>")
+				owner.dizziness += 20
+				owner.confused += 20
+				owner.Jitter(20)
+			else
+				to_chat(owner, "<span class='warning'>You feel weak and scared! If only you weren't alone...</span>")
+				owner.dizziness += 20
+				owner.confused += 20
+				owner.Jitter(20)
+				owner.adjustHalLoss(50)
+
+		if(3, 4)
+			if(!high_stress)
+				to_chat(owner, "<span class='warning'>You feel really lonely...</span>")
+			else
+				to_chat(owner, "<span class='warning'>You're going mad with loneliness!</span>")
+				owner.hallucination += 20
+
+		if(5)
+			if(!high_stress)
+				to_chat(owner, "<span class='warning'>Your heart skips a beat.</span>")
+				owner.adjustOxyLoss(8)
+			else
+				if(prob(15) && ishuman(owner))
+					var/mob/living/carbon/human/H = owner
+					var/obj/item/organ/heart/heart = H.internal_organs_by_name["heart"]
+					heart.take_damage(heart.min_bruised_damage)
+					to_chat(H, "<span class='userdanger'>You feel a stabbing pain in your heart!</span>")
+				else
+					to_chat(owner, "<span class='userdanger'>You feel your heart lurching in your chest...</span>")
+					owner.adjustOxyLoss(8)
+
+/datum/brain_trauma/severe/discoordination
+	name = "Discoordination"
+	desc = "Patient is unable to use complex tools or machinery."
+	scan_desc = "extreme discoordination"
+	gain_text = "<span class='warning'>You can barely control your hands!</span>"
+	lose_text = "<span class='notice'>You feel in control of your hands again.</span>"
+
+/datum/brain_trauma/severe/discoordination/on_gain()
+	owner.disabilities |= MONKEYLIKE
+	..()
+
+/datum/brain_trauma/severe/discoordination/on_lose()
+	owner.disabilities &= ~MONKEYLIKE
+	..()
+
+/datum/brain_trauma/severe/aphasia
+	name = "Aphasia"
+	desc = "Patient is unable to speak or understand any language."
+	scan_desc = "extensive damage to the brain's language center"
+	gain_text = "<span class='warning'>You have trouble forming words in your head...</span>"
+	lose_text = "<span class='notice'>You suddenly remember how languages work.</span>"
+	var/list/prev_languages = list()
+	var/datum/language_holder/mob_language
+
+/datum/brain_trauma/severe/aphasia/on_gain()
+	for(var/datum/language/L in owner.languages)
+		prev_languages.Add(L)
+		owner.languages.Remove(L)
+	owner.languages.Add("Gibbering")
```

Comment:
I don't think this'll work, there's a proc for this. Should also use a define for this language.